frontend/src/app/auth/auth-form/auth-form.component.ts

```diff
@@ -0,0 +1,41 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { FormControl, FormGroup, Validators } from '@angular/forms';
+import { Router } from '@angular/router';
+import { AuthService } from '../services/auth.service';
+import { Subscription } from 'rxjs';
+
+@Component({
+  selector: 'app-auth-form',
+  templateUrl: './auth-form.component.html',
+  styleUrls: ['./auth-form.component.scss'],
+})
+export class AuthFormComponent implements OnDestroy, OnInit {
+  loginForm: FormGroup = new FormGroup({
+    email: new FormControl('', [Validators.required]),
+    password: new FormControl('', [Validators.required]),
+  });
+
+  subs: Subscription = new Subscription();
+
+  constructor(private authService: AuthService, private router: Router) {}
+
+  ngOnInit() {
+    if (this.authService.isLoggedIn()) {
```

Comment:
Pls create a new guard that will protect login page for authorized users (move this line to the guard)

Reply:
You already have interceptor so you can remove this logic